tpl/template_funcs.go

```diff
@@ -739,6 +743,25 @@ func checkCondition(v, mv reflect.Value, op string) (bool, error) {
 			return !r, nil
 		}
 		return r, nil
+	case "intersect":
+		r, err := intersect(slv, slmv)
+		if err != nil {
+			return false, err
+		}
+		switch r.(type) {
+		case []string:
+			if len(r.([]string)) > 0 {
+				return true, nil
+			}
+			return false, nil
+		case []int:
```

Comment:
This is too restrictive, I think. There are are int64 and all the rest of the family.